### ISSUE TITLE
Upload failed: The server is having temporary problems.

### ISSUE THREAD

**Issue by ethanboxx** [asciinema/asciinema#355]
```
efun@inspiron ~> asciinema rec
asciinema: recording asciicast to /tmp/tmpwxmshdma-ascii.cast
asciinema: press <ctrl-d> or type "exit" when you're done
efun@inspiron:~$ echo test
test
efun@inspiron:~$ exit
exit
asciinema: recording finished
asciinema: press <enter> to upload to asciinema.org, <ctrl-c> to save locally
asciinema: upload failed: The server is having temporary problems. Try again in a minute.
asciinema: retry later by running: asciinema upload /tmp/tmpwxmshdma-ascii.cast
```
Has failed to upload every time I have tried since installing a few days ago.

OS: Ubuntu 19.04 disco
asciinema 2.0.2

**Comment by vegabook** [comment]
Same over here. Works fine on Fedora 30.

**Comment by 0xXA** [comment]
Same here

**Comment by Graph1589** [comment]
Same. OS disco too

**Comment by roaldnefs** [comment]
Currently facing the same issue on Ubuntu 19.04. Tried to install Asciinema via apt, Snap and Pip...

```bash
$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 19.04
Release:	19.04
Codename:	disco

$ uname -a
Linux laptop 5.0.0-15-generic #16-Ubuntu SMP Mon May 6 17:41:33 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux

$ asciinema --version
asciinema 2.0.2
```

**Comment by roaldnefs** [comment]
This issue is a duplicate/related to #353.

**Comment by sickill** [comment]
Solved, see the comment in #353.
